src/compiler/scanner.ts

```diff
@@ -42,6 +42,7 @@ namespace ts {
         setScriptTarget(scriptTarget: ScriptTarget): void;
         setLanguageVariant(variant: LanguageVariant): void;
         setTextPos(textPos: number): void;
+        setInJSDocType(inType: boolean): void;

```

Comment:
I think this is where you need to add `/* @internal */`

Reply:
Thanks.  I'll try that.

Reply:
Added in 64bbf5ee125b44a55cbe8efda60182ec20d815be.